Dockerfile

```diff
@@ -27,6 +27,11 @@ COPY . .
 # Uncomment the following line in case you want to disable telemetry during the build.
 # ENV NEXT_TELEMETRY_DISABLED 1
 
+RUN \
+  # Configure Corepack
+  corepack enable; \
+  corepack prepare --activate;
```

Comment:
Needed to add this statement to build the container, otherwise I kept running into build failures